index.test-d.ts

```diff
@@ -1,4 +1,4 @@
-import { expectType } from 'tsd';
+import {expectType} from 'tsd';
```

Comment:
@sindresorhus In the future, is there any suggested solution, for when my Prettier settings makes changes like these? Meaning, does everyone that contribute to your repos just create their own uncommited `.pretterirc.json` file in their own copy of a repo? Or would you ever consider allowing the addition of a `.prettierrc.json` file added to this repo (with whatever settings you'd want)?

Reply:
Only enable Prettier on projects you own or that uses Prettier. The general OSS sentiment is that you should not do unrelated changes.